tokio-util/tests/context.rs

```diff
@@ -0,0 +1,49 @@
+#![warn(rust_2018_idioms)]
+
+use tokio::{net::TcpListener, sync::oneshot};
+use tokio_util::context::TokioContext;
+
+use lazy_static::lazy_static;
+use std::future::Future;
+
+struct ThreadPool {
+    inner: futures::executor::ThreadPool,
+    rt: tokio::runtime::Runtime,
+}
+
+lazy_static! {
```

Comment:
We don't need to do the lazy static trick this is only for ease of use. So I would likely remove this and just pass the executor in manually for tests.